src/arb/acb.jl

```diff
@@ -96,6 +119,14 @@ function convert(::Type{ComplexF64}, x::AcbFieldElem)
   return complex(v, w)
 end
 
+function convert(::Type{ComplexF64}, x::AcbFieldElem)
+  return ComplexF64(x)
+end
+
+function convert(::Type{Float64}, x::AcbFieldElem)
+  return Float64(x)
+end
```

Comment:
@thofma This `convert` method was added "for symmetry". But of course we can also remove it and only keep the `convert` to `ComplexF64` (to retain compatibility), or drop both...